configuration/conf-files/tests/oso-clusters-with-removed-clusters.conf

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
in this file you removed a single cluster (`1a`), in the middle of the list. Could you remove the first and last entries as well, to test all cases?

Reply:
Done in https://github.com/fabric8-services/fabric8-cluster/pull/82/commits/5749684d6b8f492238793983724508850e52fec5

Reply:
thanks @alexeykazakov!